skrf/noisyComponents/__init__.py

```diff
@@ -0,0 +1,9 @@
+
+
+
+from .attenuator import *
+from .circulator import *
+from .rlc_series_2port import *
+from .rlc_shunt_2port import *
+from .bjt_ce_2port import *
+from .wilkinson_nport import *
```

Comment:
I would recommand to postpone the `noisyComponents ` in a future PR, the time for `noisyNetwork `API to be integrated. 

On the API point of view, we need to discuss (@arsenovic ?). I see two things:
- does it have sense to have a bunch of new classe names, but knowing that these classes are in fact all `Network ` or `NoisyNetwork `objects. 
- If there is an interest in doing so, we could be more ambitious: Instead of a directory `noisyComponents/`, maybe it is worth creating a `components/` directory, with some noisy components inside (or as a subdir)

Reply:
What I can do (at least temporarily) is place noisyComponents within the doc/examples/noisynetworks. That way all our examples will work, but we don't have to make noisyComponents a part of the core scikit-rf code. Does this sound good to you?

As far as adding new class names, I believe this is within the spirit of object oriented programing. It is true that all of these new classes are Networks or NoisyNetworks, in the same way that a dog or a cat is an animal. For example, you might make an animal class have 4 legs, a mouth and ears, but the derived dog class might add the ability to bark and the derived cat class have whiskers. An attenuator and a circulator are both Networks, so the natural object oriented mechanism for creating these devices in code is to derived them from Network using inheritance. 

However, that's more philosophy than anything. You could of course just create a Network and name it an attenuator.  Since these decisions are more up to you and the other creators of scikit-rf, I will happily defer to your judgement.

As far as an opinion goes, I think it would be nice to have a folder called components where all objects are derived from NoisyNetwork. Since NoisyNetwork just extends Network, all the functionality that you would expect from Network would be transferred to the derived component.